### DISCUSSION TITLE
New Icon for PTerm

### DISCUSSION THREAD

**Discussion by MarvinJWendt** [pterm/pterm#71] [Ideas]
This is the chat, where we can discuss the new icon of PTerm.

@nshem thanks for your submission, we can continue to chat here 😄

**Comment by nshem** [comment]
hey, sorry for the delay. here are 4 options.
i can make changes in everyone of them.
they were made to be clear both as icons - about 20*20 px, and as logos - 100*100.
so i would advise to shrink them to those sizes when viewing.

![go1](https://user-images.githubusercontent.com/20145695/97102205-4ef81800-16ac-11eb-97e8-56c230771b28.png)
![pterm1](https://user-images.githubusercontent.com/20145695/97102206-50294500-16ac-11eb-8069-ab3732c3876d.png)
![pterm2](https://user-images.githubusercontent.com/20145695/97102207-50294500-16ac-11eb-9e77-6fa9bab903dd.png)
![pterm-full](https://user-images.githubusercontent.com/20145695/97102208-50c1db80-16ac-11eb-982c-d3546b6a21cd.png)
@MarvinJWendt

**Reply by MarvinJWendt** [reply]
Hi @nshem, I thought about a more modern logo. Best option would be an SVG file, so it won't be pixelated, when zoomed in. Most likely with a transparent background. The logo should also show what PTerm is/does. For example a decorated terminal. I thought about creating a kind of word mark with a modern terminal font.
